cookbooks/aws-parallelcluster-install/resources/install_packages/partial/_install_packages_rhel_amazon.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+action :install_base_packages do
+  package node['cluster']['base_packages'] do
+    retries 10
+    retry_delay 5
+    flush_cache({ before: true })
```

Comment:
What happens if you flush cache on debian?

Reply:
The implementation do not allow that argument returning
`NoMethodError: undefined method `flush_cache' for Chef::Resource::AptPackage`